### PR TITLE
feat(mnq): add nats and sqs beta resources

### PR DESCRIPTION
**Pull request by Codelax**

### CODE REVIEW

File: scaleway/resource_mnq_sqs_test.go [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+					testAccCheckScalewayMNQSQSExists(tt, "scaleway_mnq_sqs.main"),
+					testCheckResourceAttrUUID("scaleway_mnq_sqs.main", "id"),
+					resource.TestCheckResourceAttrSet("scaleway_mnq_sqs.main", "endpoint"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccScalewayMNQSQS_AlreadyActivated(t *testing.T) {
```
Comment (remyleone) [inline scaleway/resource_mnq_sqs_test.go:80]: Could you add a test that test whether I can activate SQS and Nats in a different region than the one specified in the provider?

File: scaleway/resource_mnq_sqs_queue.go [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+				Sensitive:   true,
+				Description: "SQS secret key",
+			},
+			"fifo_queue": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Whether the queue is a FIFO queue. If true, the queue name must end with .fifo",
+			},
+			"content_based_deduplication": {
```
Comment (remyleone) [inline scaleway/resource_mnq_sqs_queue.go:72]: Could you add a description here?

File: docs/resources/mnq_sqs_queue.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    can_manage = false
+    can_receive = true
+    can_publish = false
+  }
+}
+
+resource scaleway_mnq_sqs_queue main {
+  project_id = scaleway_mnq_sqs.main.project_id
+  name = "my-queue"
+  endpoint = scaleway_mnq_sqs.main.endpoint
```
Comment (remyleone) [inline docs/resources/mnq_sqs_queue.md:33]: Could endpoint be removed? I'm not sure to understand why I need to specify it

File: docs/resources/mnq_sqs.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Examples
+
+### Basic
+
+```hcl
+// For default project
+resource "scaleway_mnq_sqs" "main" {}
+
+// For specific project
```
Comment (remyleone) [inline docs/resources/mnq_sqs.md:20]: add "in the default region"

File: docs/resources/mnq_nats_credentials.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+For further information please check
+our [documentation](https://www.scaleway.com/en/docs/serverless/messaging/reference-content/nats-overview/)
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_mnq_nats_credentials" "main" {
+  name = "nats-account"
```
Comment (remyleone) [inline docs/resources/mnq_nats_credentials.md:18]: Should the account be specified here?

File: scaleway/resource_mnq_sqs.go [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+		Schema: map[string]*schema.Schema{
+			"endpoint": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Endpoint of the SQS service",
+			},
+			"region":     regionSchema(),
+			"project_id": projectIDSchema(),
+		},
+		/* Goal here is to ForceNew if Status is found to be disabled, maybe just add a ForceNew `status` field in schema
```
Comment (remyleone) [inline scaleway/resource_mnq_sqs.go:30]: Should this code be included or removed?

File: scaleway/resource_mnq_sqs_test.go (hunk 1 above)
Comment (Codelax) [reply scaleway/resource_mnq_sqs_test.go:80]: Only fr-par is currently available

File: docs/resources/mnq_sqs_queue.md (hunk 3 above)
Comment (Codelax) [reply docs/resources/mnq_sqs_queue.md:33]: Added a test without endpoint specified

**Review by remyleone — APPROVED** [review]

**Pull request merged**